components/prism-sas.js

```diff
@@ -1,3 +1,16 @@
+(function (Prism) {
+
+	numberpattern = /\b(?:\d[\da-f]*x|\d+(?:\.\d+)?(?:e[+-]?\d+)?)\b/i;
```

Comment:
Please make it `var numberPattern = ...`.

We are using the IEFE because we don't want global variables and camel case greatly improves readability.

Reply:
Will do.